test/spec.yml

```diff
@@ -1054,3 +1055,21 @@
             90%  * ms*
             95%  * ms*
             99%  * ms*
+
+- test: empty env
+  steps:
+    -   in: bn create node8 emptyenv
+    -   in: |-
+            cat <<EOF >>/home/dockeruser/test/binaris.yml
+                env:
+                    nosuchenv:
+            EOF
+    -   in: bn deploy emptyenv
+        out: |-
+            Ignoring non existing env var 'nosuchenv'*
```

Comment:
Nit: use `%` here (restricts to single line, added in #398).

Reply:
unfortunately, there are multiple lines there (all the deploy blurb)